### PR TITLE
Change backend to http

### PR DESCRIPTION
**Pull request by eanker**
This ensures the proxy works.
It has no security implications due to the proxy

---
Review this pull request [on Preview Code](https://preview-code.com/projects/preview-code/backend/pulls/17/overview).

### CODE REVIEW

File: src/main/resources/jetty.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
     <!-- for all configuration that may be set here.                 -->
     <!-- =========================================================== -->
     <New id="httpConfig" class="org.eclipse.jetty.server.HttpConfiguration">
-      <Set name="secureScheme">https</Set>
+      <Set name="secureScheme">http</Set>
       <Set name="securePort">8443</Set>
```
Comment (txsmith) [inline src/main/resources/jetty.xml]: I believe the two lines above should be removed entirely as we do not support secure connections to the backend at all anymore.

Comment (txsmith) [inline src/main/resources/jetty.xml]: The docs indicate this here: http://download.eclipse.org/jetty/stable-9/apidocs/org/eclipse/jetty/server/HttpConfiguration.html#getSecurePort--

**Comment by eanker** [comment]
Fixed

**Comment by txsmith** [comment]
Works! :+1:

**Pull request merged**